apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/JobsScheduler.java

```diff
@@ -65,6 +65,7 @@
 public class JobsScheduler {
   private static final int TASKS_WAIT_TIMEOUT_HOURS = 12;
   private static final int DEFAULT_MAX_NUM_CONCURRENT_JOBS = 40;
+  private static final int DEFAULT_TABLE_CREATION_CUTOFF_HOUR = 72;
```

Comment:
is this from heuristic or any reason to reach this value ? 
Any compliance related indication for this ?

Reply:
72 hour is based on minimum hours before which snapshots start getting expired. 
Even in current state, any table which is newer than 72 hour, will not have snapshots older than 72 huors and SE and OFD will have not effect on it.
I do not see compliance changes related to it.

Will need to pass custom param for `cutoffHours` for retention job only. Will work on it after this is merged.

Reply:
I think it is OK to leave retention job alone ? Since that requires explicit info to be provided from users. Remember the use case for this PR's change is to avoid wasting computation on those staging tables.